### ISSUE TITLE
Sigmoid activation 

### ISSUE THREAD

**Issue by ulkursln** [hfawaz/ijcnn19attacks#2]
Hi, I am trying to use MSD function for custom ResNet  trained with timeseries. It is a binary classification task, and the activation is sigmoid instead of softmax. Program throws following error:
 File "/home/../git/ijcnn19attacks/src/cleverhans_copy/utils_keras.py", line 132, in _get_softmax_name
    raise Exception("No softmax layers found")
Exception: No softmax layers found

Is softmax activation mandatory, or just updating "softmax" with "sigmoid" in all functions of utils_keras.py enough?

**Comment by ulkursln** [comment]
By the way, since MDS function requires attacked dataset, I firstly run "python3 main.py attack fgsm " and that is when I got the error

**Comment by hfawaz** [comment]
Hi, 
Thanks for your interest. 
I believe that you should change the underlying code in order to use sigmoid. 
I would recommend to just have a softmax layer with two neurons in order to avoid modifying the code.
